install.sh

```diff
@@ -1,6 +1,14 @@
-#!/bin/bash
+#!/bin/bash -e
 
 dir=~/.local/share/gnome-shell/extensions/vpn-indicator@howdoicomputer.fastmail.com
 
-rm -rf $dir
+if [ -d $dir ]; then
+  rm -rf $dir
+else
+  echo "brand-new install..."
+fi
+mkdir -p $dir
```

Comment:
the `-p` option here is equivalent to an `install` command, and ensures that the mkdir command does not fail out if a directory already exists.